modules/plot/main/coffee/charts/PieChart.coffee

```diff
@@ -79,7 +81,7 @@ class PieChart extends hx.EventEmitter
 
   render: ->
     self = this
-    selection = hx.select(@selector)
+    selection = hx.select(@selector).class('hx-pie-chart')
```

Comment:
Give it 100% height and width - this is for pie charts that appear in detached elements that get added to the dom

Otherwise width and height are 0 below